_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -839,21 +882,20 @@ Figure 2. Relationship Model between the different software elements (part 2).
 
 ![An UML diagram representing the hierarchical relation of an IT Solution and its components. There are 5 rectangles on the diagram connected by relationship lines. The first rectangle to the left is titled IT Solution. It is connected to a second rectangle, titled IT Product, located on its right by a relationship line; the end of the line starting from the IT Solution rectangle has a 1 and the other end has 1... and a small white diamond. The IT Product rectangle is connected to a third rectangle, titled Software, located on its right by a relationship line; the end of the line starting from the IT Product rectangle has a 1 and the other end has a 1... and a small white diamond. The Software rectangle has two relationships connections. The first connection is to a rectangle, titled Configuration, located below the Software rectangle; the end of the line starting from the Software rectangle has a 1 and the other end has a 1... and a small black diamond. The second connection is to another rectangle, titled Infrastructure, located to the right of the Software rectangle; the end of the line starting from the Software rectangle has a 1 and the other end has a 1... and a small white diamond. There is one last connection,linking the Infrastructure and Configuration rectangles; the end of the line starting from the Infrastructure rectangle has a 1... and the other end has a 1... and a small black diamond.]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
 
-**IT Project**
+**IT-Enabled Project**
 
 A temporary endeavour undertaken to create a unique technology product, service, or result.
-The temporary nature of IT projects indicates that an IT project has a definite beginning and end.
+The temporary nature of IT-Enabled projects indicates that it has a definite beginning and end.
```

Comment:
Because 'projects' is plural I would change this to ...."...indicates that they have a definite...."  Or instead "..indicates that there is a definite...."